### PR TITLE
Issue 15 - Utility 2 - Add private browsing highlighter functionality to index.js

### PR DESCRIPTION
**Pull request by FrancescoSTL**
This PR adds the switch case for highlighting the private browsing button, as well as the function needed to open the main menu.

Issue ref: https://github.com/mozilla/all-aboard/issues/15

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -166,6 +166,23 @@ function showBadge() {
 }
 
 /**
+ * Purpose: Highlight a given item in the browser chrome
+ * @param {string} item - String of item you are attempting to highlight
```
Comment (schalkneethling) [inline index.js]: Nit: Perhaps rephrase this as "Item you wish to highlight's name as a string" ?

File: index.js [hunk 2]
```diff
@@ -166,6 +166,23 @@ function showBadge() {
 }
 
 /**
+ * Purpose: Highlight a given item in the browser chrome
```
Comment (schalkneethling) [inline index.js]: Nit: I reckon we do not need to prepend "Purpose" as it is implied

File: index.js [hunk 3]
```diff
@@ -166,6 +166,23 @@ function showBadge() {
...
+ * @param {string} item - String of item you are attempting to highlight
+ */
+function highLight(item) {
+    var activeWindow = utils.getMostRecentBrowserWindow();
+        
+    UITour.getTarget(activeWindow, item, false).then(function(chosenItem) {
+        try {
+            UITour.showHighlight(activeWindow, chosenItem, "wobble");
```
Comment (schalkneethling) [inline index.js]: Use single quotes here.

File: index.js [hunk 4]
```diff
@@ -166,6 +166,23 @@ function showBadge() {
...
+        }
+        catch(e) {
+            console.log("Could not highlight element. Check if UITour.jsm supports highlighting of element passed.");
```
Comment (schalkneethling) [inline index.js]: Use single quotes here.

File: index.js [hunk 5]
```diff
@@ -166,6 +166,23 @@ function showBadge() {
 }
 
 /**
+ * Purpose: Highlight a given item in the browser chrome
+ * @param {string} item - String of item you are attempting to highlight
+ */
+function highLight(item) {
+    var activeWindow = utils.getMostRecentBrowserWindow();
```
Comment (schalkneethling) [inline index.js]: `let` for consistency?

File: index.js [hunk 6]
```diff
@@ -166,6 +166,23 @@ function showBadge() {
...
+        
+    UITour.getTarget(activeWindow, item, false).then(function(chosenItem) {
+        try {
+            UITour.showHighlight(activeWindow, chosenItem, "wobble");
+        }
+        catch(e) {
```
Comment (schalkneethling) [inline index.js]: `e` is defined here but never used. You could do `console.log('Could not highlight.....', e);`

**Comment by schalkneethling** [comment]
Couple of small items, but looks good.

**Comment by FrancescoSTL** [comment]
Ready for merge 💯

**Comment by schalkneethling** [comment]
@FrancescoSTL needs a rebase with master, other than that r+

**Comment by FrancescoSTL** [comment]
Rebase complete

**Comment by schalkneethling** [comment]
r+
